tests/unit/suites/plugins/filesystem/local/adapter/MediaFileAdapterLocalTest.php

```diff
@@ -359,4 +359,131 @@ public function testDeleteInvalidPath()
 		// Fetch the file from the root folder
 		$adapter->delete('invalid');
 	}
+
+	public function testCopy()
```

Comment:
Please split individual tests into their own function. Every test function should do only one test like copy file or copy folder.